v1.1-events.yml

```diff
@@ -52,36 +53,29 @@ paths:
         '404':
           $ref: '#/responses/NotFound'
 
-  /school_admins/{id}/events:
+  /events/{id}:
     get:
-      description: Returns a list of events for a school admin
-      operationId: getEventsForSchoolAdmin
+      tags: ["Events"]
+      description: Returns the specific event
+      operationId: getEvent
       parameters:
       - in: path
         name: id
         required: true
         type: string
-      - in: query
-        name: limit
-        type: integer
-      - in: query
-        name: starting_after
-        type: string
-      - in: query
-        name: ending_before
-        type: string
       responses:
         '200':
           description: OK Response
           schema:
-            $ref: '#/definitions/EventsResponse'
+            $ref: '#/definitions/EventResponse'
         '404':
           $ref: '#/responses/NotFound'
 
-  /schools/{id}/events:
+  /teachers/{id}/events:
     get:
-      description: Returns a list of events for a school
-      operationId: getEventsForSchool
+      tags: ["Record-specific Events"]
```

Comment:
That was Amelie's suggestion but she wanted to chew on it for a while. I figured it was a simple naming change later on